src/main/java/structures/Arrayer.java

```diff
@@ -22,21 +44,39 @@ public static int[] evensOnly(int[] input) {
      * ex: [1004, 9181, 1700, 4565] -> [4, 1, 0, 5]
      */
     public static int[] lastOfFourDigits(int[] input) {
-        // TODO implement lastOfFourDigits()
-        return new int[0];
+        // creates an array called result that is equal to the length of input
+        int[] result = new int[input.length];
+        // a for loop that goes through each position in input
+        for (int i = 0; i<input.length; i++) {
+            // finds the last number and adds it to result
+            result[i] = input[i]%10;
```

Comment:
be careful here - should be i < input.length and input[i] % 10 (notice spacing) 👀